### ISSUE TITLE
Ensemble Analysis for Steady State Problems

### ISSUE THREAD

**Issue by QiyaoWei** [SciML/DifferentialEquations.jl#797]
Dear authors,

Hope this message finds you well. I am writing to raise a question about Ensemble analysis, documented here (https://diffeq.sciml.ai/stable/features/ensemble/). I understand that it ensemble analysis works for ODE problems with a timetrace solution---is there similar ensemble analysis support for steady state problems (where the final solution is not over a time interval but just a final value)? If so, are there tutorial examples? Thanks a lot!

**Comment by ChrisRackauckas** [comment]
Yeah, I think this is an oversight. The ensemble stuff all assumes a timeseries and we need some extra dispatches.
